src/runtime/particle-execution-host.ts

```diff
@@ -193,6 +194,16 @@ class PECOuterPortImpl extends PECOuterPort {
     this.SimpleCallback(callback, res);
   }
 
+  async onBackingProxyMessage(store: BackingStore<CRDTTypeRecord>, message: ProxyMessage<CRDTTypeRecord>, entityId: string, callback: number) {
+    if (!(store instanceof BackingStore)) {
+      this.onReportExceptionInHost(new SystemException(new Error('expected new-style store but found old-style store hooked up to new stack'), 'onProxyMessage', ''));
```

Comment:
Suggestion: something simpler, like: `Expected BackingStore for onBackingProxyMessage`